gensim/models/wrappers/ldavowpalwabbit.py

```diff
@@ -235,6 +235,14 @@ def log_perplexity(self, chunk):
                  corpus_words)
         return bound
 
+    def get_topics(self):
```

Comment:
The common get_topics method for all TM looks nice, can you add this method to base TM class?